src/pfe/Dockerfile_x86_64

```diff
@@ -148,8 +148,10 @@ COPY /portal/NOTICE.md /
 WORKDIR /portal
 ARG NODE_ENV
 ARG IMAGE_BUILD_TIME
+ARG ENABLE_CODE_COVERAGE
```

Comment:
Would it clear up some of the code below in `src/pfe/build` if we used:
```
ARG ENABLE_CODE_COVERAGE=false
```
to set a default value?

Reply:
Have to keep it as is due to the `--build-arg` being passed in overwriting the defaults in the Dockerfile. I've added the default anyway so in the future if we build the Dockerfile without the `--build-arg` flag we won't error out.